packages/lisk-cryptography/src/encrypt.js

```diff
@@ -71,16 +72,20 @@ export const decryptMessageWithPassphrase = (
 	const nonceBytes = hexToBuffer(nonce);
 
 	try {
-		const decoded = nacl.box.open(
+		const decoded = openBox(
 			cipherBytes,
 			nonceBytes,
 			convertedPublicKey,
 			convertedPrivateKey,
 		);
 		return Buffer.from(decoded).toString();
 	} catch (error) {
-		if (error.message.match(/bad nonce size/)) {
-			throw new Error('Expected 24-byte nonce but got length 1.');
+		if (
+			error.message.match(
+				/bad nonce size|nonce must be a buffer of size crypto_box_NONCEBYTES/,
+			)
+		) {
+			throw new Error('Nonce must be a buffer of size 24.');
```

Comment:
This isn't true: we convert from a hex string. How about `Expected nonce to be 24 bytes.`?